### ISSUE TITLE
Why collect data from your users?

### ISSUE THREAD

**Issue by NikolaiT** [alexandrevicenzi/Flex#158]
Why do collect data from your users in the file `Flex/templates/partial/piwik.html`?

```
<noscript><p><img src="//bnjbvr.alwaysdata.net/piwik/piwik.php?idsite=1" style="border:0;" alt="" /></p></noscript>
```

**Comment by alorence** [comment]
This has been addressed in #94.

See the file in mster branch: https://github.com/alexandrevicenzi/Flex/blob/master/templates/partial/piwik.html

```
<noscript><p><img src="//{{ PIWIK_URL }}/piwik.php?idsite={{ PIWIK_SITE_ID }}" style="border:0" alt="" /></p></noscript>
```

**Comment by alexandrevicenzi** [comment]
@NikolaiT update your Flex version to the latest one.
